pkg/hive/resources.go

```diff
@@ -193,6 +193,11 @@ func adoptedClusterDeployment(namespace, clusterName, clusterID, infraID, locati
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ClusterDeploymentName,
 			Namespace: namespace,
+			Annotations: map[string]string{
+				// https://github.com/openshift/hive/pull/2501
+				// Disable hibernation controller as it is not used as part of ARO's Hive implementation 
+				"hive.openshift.io/infra-disabled": "true",
```

Comment:
Super nit: we could make a `const` for this in `pkg/hive` since it's used in two places.

Reply:
addressed in latest commit